### PR TITLE
Updated valid path parts

### PR DESCRIPTION
**Pull request by antonin-francois**

### CODE REVIEW

**Comment by antonin-francois** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/60?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#60** <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/60?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#59** <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/59?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#58** <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/58?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#57** <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/57?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#56** <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/56?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#55** <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/55?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#54** <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/54?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#53** <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/53?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @antonin-francois and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by walerowicz-miam — APPROVED** [review]

File: mealz-shared-analytics/src/commonMain/kotlin/mealz/ai/AbstractSharedAnalytics.kt [hunk 1]
```diff
@@ -37,8 +37,9 @@ abstract class AbstractSharedAnalytics {
 
         // Only keep Mealz's part
         val pathSplit = path.substringAfter("/miam/").splitToPlatformList('/')
-        val validParts = PlatformList("", "recipes", "liked", "categories", "my-meals", "detail", "replace-item", "sponsor",
-            "meals-planner", "catalog", "results", "basket-preview", "finalize", "onboarding", "locator")
+        val validParts = PlatformList("", "recipes", "products", "liked", "categories", "basket", "preferences", "item-selector",
```
Comment (diarmuidmcg) [inline mealz-shared-analytics/src/commonMain/kotlin/mealz/ai/AbstractSharedAnalytics.kt:40]: why isnt this alphabetazied?

File: mealz-shared-analytics/src/commonMain/kotlin/mealz/ai/AbstractSharedAnalytics.kt [hunk 2]
```diff
@@ -37,8 +37,9 @@ abstract class AbstractSharedAnalytics {
...
+            "shopping", "cooking", "ordered",  "sponsor", "catalog", "meals-planner", "results", "basket-preview", "finalize", "onboarding",
```
[the remark this answers is not in the document]
Comment (antonin-francois) [reply mealz-shared-analytics/src/commonMain/kotlin/mealz/ai/AbstractSharedAnalytics.kt:41]: recipes is the home (all recipes) or for a specific recipe (/miam/recipes/ID)
catalog is the meals-planner's catalog
categories is for a specific category (/miam/categories/ID)

File: mealz-shared-analytics/src/commonMain/kotlin/mealz/ai/AbstractSharedAnalytics.kt [hunk 3]
```diff
@@ -37,8 +37,9 @@ abstract class AbstractSharedAnalytics {
...
+            "locator", "orders")
```
[the remark this answers is not in the document]
Comment (antonin-francois) [reply mealz-shared-analytics/src/commonMain/kotlin/mealz/ai/AbstractSharedAnalytics.kt:42]: yes

**Review by diarmuidmcg — APPROVED** [review]

**Pull request merged**